### PR TITLE
hide login form if db auth is disabled

### PR DESCRIPTION
**Pull request by petrjasek**
SDNTB-626

### CODE REVIEW

File: scripts/core/auth/login-modal.html [hunk 1]
```diff
@@ -9,12 +10,12 @@
         <div class="form-handler" ng-show="!changePassword">
             <figure class="avatar" style="background-image:url('{{ identity.Avatar.href || 'images/avatar_200.png' }}')" ng-show="identity.Avatar"></figure>
             <form name="loginForm" ng-submit="authenticate()">
-                <fieldset class="inputs">
+                <fieldset class="inputs" ng-show="features.auth_db">
```
Comment (tomaskikutis) [inline scripts/core/auth/login-modal.html:13]: Doesn't `ng-if` work here?

Comment (petrjasek) [reply scripts/core/auth/login-modal.html:13]: no it doesn't, it's angular thing, the form it sends then is empty

Comment (pablopunk) [reply scripts/core/auth/login-modal.html:13]: I think ng-if creates a new child scope for everything inside it, that's why

**Comment by petrjasek** [comment]
will be in 1.33.2 and 1.34 eventually

**Pull request merged**